ara/templates/playbook.html

```diff
@@ -39,11 +39,11 @@
     {% for stat in playbook.stats %}
     <tr>
       <td>{{ make_link('playbook_host', stat.host.name, playbook=playbook.id, host=stat.host.name) }}</td>
-      <td>{{ stat.ok }}</td>
-      <td>{{ stat.changed }}</td>
-      <td>{{ stat.failed }}</td>
-      <td>{{ stat.skipped }}</td>
-      <td>{{ stat.unreachable }}</td>
+      <td>{% if stat.ok >= 1 %}{{ make_link('playbook_host', stat.ok, playbook=playbook.id, host=stat.host.name, status='ok') }}{% else %}{{ stat.ok }}{% endif %}</td>
```

Comment:
Too much boilerplate code!  If nothing else we should use a jinja2 macro for this.